Makefile

```diff
@@ -0,0 +1,26 @@
+build-dev:
```

Comment:
The way you are using `make` here like a command runner is fine but make has a lot of tricks under its sleeve that we can exploit later. Just a small addition for now, the targets here(the "tasks" names) are usually expected to be something that (will)exist in the file system(the set of commands should create said target when the file is not found), when the target is not a file is said to be a [Phony target](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html) that should be marked as one with `.PHONY: target-1 target-2 ...` so make wont try to find them in the file system. One line at the top with all the target names is fine ;)

Reply:
fixed in https://github.com/valibre-org/vln-node/pull/76/commits/a0c128f37392d07cd666087b274282189b1d2704